lib/application.js

```diff
@@ -119,6 +119,7 @@ Box.Application = (function() {
 		globalConfig = {};
 		modules = {};
 		services = {};
+		serviceStack = [];
```

Comment:
No, I actually had to add this because when running the test suite through leche it was failing on subsequent runs and throwing circular initialization errors. Once I added clearing the stack to reset, it ceased.